app/models.py

```diff
@@ -2068,6 +2068,14 @@ class FactNotificationStatus(db.Model):
     updated_at = db.Column(db.DateTime, nullable=True, onupdate=datetime.datetime.utcnow)
 
 
+class FactProcessingTime(db.Model):
+    __tablename__ = "ft_processing_time"
+
+    bst_date = db.Column(db.Date, index=True, primary_key=True, nullable=False)
+    messages_total = db.Column(db.Integer(), nullable=False)
+    messages_within_10_secs = db.Column(db.Integer(), nullable=False)
```

Comment:
If we add more columns to further breakdown the data, it would give us more information. However, would we use it? 
We would not have any of the historical data, we can only get a breakdown as far back as September 2019, that would take some time to collect that data from the notification_history table.

Reply:
After a quick chat with the team, we've decided to leave the aggregation as is.